src/renderer/components/+helm-charts/helm-chart-details.tsx

```diff
@@ -46,6 +46,7 @@ interface Dependencies {
 
 @observer
 class NonInjectedHelmChartDetails extends Component<HelmChartDetailsProps & Dependencies> {
+  private readonly imgLoadingFailed = observable.set<string>(); // The IDs of the HelmChart instances
```

Comment:
A code readability comment: the name of the variables should be self-descriptive. There is something wrong with it if we have such a comment afterwards.

How about that?

```
private readonly helmChartsWithFailedImagesLoad = ...
```